### PR TITLE
new hue motion 9290030675 and 9290030674 have 5 levels of sentivity

### PR DESCRIPTION
**Pull request by MattL0**

### CODE REVIEW

**Comment by MattL0** [comment]
Please check , i have poor programming skills. 

Only added 2 level of sensitivity the the newer hue motion sensors

File: devices/philips.js [hunk 1]
```diff
@@ -2019,15 +2019,15 @@ module.exports = [
...
-            exposes.enum('motion_sensitivity', ea.ALL, ['low', 'medium', 'high']),
+            exposes.enum('motion_sensitivity', ea.ALL, ['very low', 'low', 'medium', 'high', 'very high']),
             exposes.binary('led_indication', ea.ALL, true, false).withDescription('Blink green LED on motion detection'),
             exposes.numeric('occupancy_timeout', ea.ALL).withUnit('second').withValueMin(0).withValueMax(65535)],
-        toZigbee: [tz.occupancy_timeout, tz.hue_motion_sensitivity, tz.hue_motion_led_indication],
+        toZigbee: [tz.occupancy_timeout, tz.hue_motion_sensitivity_2, tz.hue_motion_led_indication],
         endpoint: (device) => {
             return {
-                'default': 2, // default
+                'default': 4, // default
```
Comment (Koenkk) [inline devices/philips.js:2030]: Can you confirm this does not break setting the `occupancy_timeout` and `led_indication`?

**Comment by MattL0** [comment]
Thanks i'll test and make another pr

File: devices/philips.js (hunk 1 above)
Comment (MattL0) [reply devices/philips.js:2030]: Adding endpoint 4 was an error. I thought it was value by default. So yes it broke the device.

**Pull request closed without merging**